docs/gettingstarted/quickstart.rst

```diff
@@ -0,0 +1,268 @@
+:layout: homepage
```

Comment:
I had Sphinx complaining at one point (as a warning, but still complaining) about something not being included in any toctree. Possibly if docs/index has gettingstarted/index in its toctree, quickstart.rst can be renamed gettingstarted/index.rst and that problem won't happen.